lib/script/allscripts.php

```diff
@@ -38,6 +40,53 @@
             echo "<script type=\"text/javascript\" src=\"lib/script/".$File."?v=".$gVersion."\"></script>\n";
         }
     }
+    elseif (PHP_SAPI == "cli")
```

Comment:
Ah - now I see what you meant.
Please no tooling in this area. This should be moved to an external script, e.g. Grunt file or sth. like that.